static/app/components/profiling/flamegraph/flamegraphSpanTooltip.tsx

```diff
@@ -67,10 +67,7 @@ export function FlamegraphSpanTooltip({
           backgroundColor={formatColorForSpan(hoveredNode, spansRenderer)}
         />
         {spanChart.formatter(hoveredNode.duration)}{' '}
-        {formatWeightToTransactionDuration(hoveredNode, spanChart)}{' '}
-        {hoveredNode.node.span.op === 'missing instrumentation'
-          ? t('Missing instrumentation')
-          : hoveredNode.node.span.description}
+        {formatWeightToTransactionDuration(hoveredNode, spanChart)} {hoveredNode.text}
```

Comment:
I think you'll lose the whitespace between the duration and the text here right?

Reply:
Doesnt look like it, but I'm always confused by how this sometimes works and how it doesnt 😓 

<img width="377" alt="CleanShot 2023-01-11 at 17 00 07@2x" src="https://user-images.githubusercontent.com/9317857/211927011-5c01e3a5-99b2-45b8-9dbc-eec0b2d56bb8.png">